web/src/features/monitoring-observers/components/MonitoringObserverDetails/MonitoringObserverDetails.tsx

```diff
@@ -1,24 +1,37 @@
-import { Outlet, useLoaderData, useNavigate } from '@tanstack/react-router';
-import { MonitoringObserver } from '../../models/MonitoringObserver';
+import { PencilIcon } from '@heroicons/react/24/outline';
+import { useLoaderData, useNavigate } from '@tanstack/react-router';
+import type { FunctionComponent } from '@/common/types';
 import Layout from '@/components/layout/Layout';
-import { Card, CardContent, CardDescription, CardFooter, CardHeader, CardTitle } from '@/components/ui/card';
-import { Tabs, TabsContent, TabsList, TabsTrigger } from '@/components/ui/tabs';
+import TableTagList from '@/components/table-tag-list/TableTagList';
 import { Badge } from '@/components/ui/badge';
-import { Separator } from '@/components/ui/separator';
 import { Button } from '@/components/ui/button';
-import { PencilIcon } from '@heroicons/react/24/outline';
-import TableTagList from '@/components/table-tag-list/TableTagList';
+import { Card, CardContent, CardFooter, CardHeader, CardTitle } from '@/components/ui/card';
+import { Separator } from '@/components/ui/separator';
+import { Tabs, TabsContent, TabsList, TabsTrigger } from '@/components/ui/tabs';
+import type { MonitoringObserver } from '../../models/MonitoringObserver';
+import { MonitoringObserverForms } from '../MonitoringObserverForms/MonitoringObserverForms';
 
-export default function MonitoringObserverDetails() {
+export default function MonitoringObserverDetails(): FunctionComponent {
   const observer: MonitoringObserver = useLoaderData({ strict: false });
   const navigate = useNavigate();
-  const navigateToEdit = () => {
-    navigate({ to: '/monitoring-observers/$monitoringObserverId/edit', params: { monitoringObserverId: observer.id } });
+  const navigateToEdit = (): void => {
+    void navigate({
+      to: '/monitoring-observers/$monitoringObserverId/edit',
+      params: { monitoringObserverId: observer.id },
+    });
   };
 
   return (
     <Layout title={`${observer.firstName} ${observer.lastName}`}>
-      <Tabs defaultValue='observer-details'>
+      <Tabs
+        defaultValue='observer-details'
+        onValueChange={(tab) => {
+          void navigate({
```

Comment:
this is neat, can we have something similar for the form answers dropdown as well ?
![image](https://github.com/commitglobal/votemonitor/assets/1877190/423c4f3a-0ac3-4669-9826-df32cadfe7af)

Reply:
we can descope it and implement it another time